### DISCUSSION TITLE
Welcome to machinekit-hal Discussions!

### DISCUSSION THREAD

**Discussion by the-snowwhite** [machinekit/machinekit-hal#340] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BjarneSJ** [comment]
I want to Install Machinekit-Hal to use Terasic DE10 Standard to control a five axis CNC-Machine. Debian is the prefered OS. 

Is it possible to get the Machinekit-Hal on a SD card to be used on the Terasic DE10 Standard?

Bjarne

**Reply by BjarneSJ** [reply]
Hello Machinekit community,

 I am a retired individual with a long-standing interest in programming as a hobby, spanning several decades.

Currently, I am using Ubuntu 24.04 and Debian 12.
